### ISSUE TITLE
Invalid space in pseudo-class selector not detected

### ISSUE THREAD

**Issue by Famlam** [CSSLint/csslint#451]
``` CSS
div: hover {
  display: none;
}
```

The code above is broken (as there should be no space between `:` and `hover`), but CSSLint doesn't detect this.
